kubeflow/core/tests/tf-job_test.jsonnet

```diff
@@ -1,12 +1,29 @@
 local tfjob = import "../tf-job-operator.libsonnet";
-local params = {
-  namespace:: "test-kf-001",
+local paramsv1alpha1 = {
+  name:: "tf-job-operator",
+  tfJobImage:: "gcr.io/kubeflow-images-public/tf_operator:v20180226-403",
```

Comment:
Please change to 
gcr.io/kubeflow-images-public/tf_operator:v0.3.0 so it won't collide with #1608